Composer/packages/client/src/recoilModel/atoms/appState.ts

```diff
@@ -369,3 +369,33 @@ export const showGetStartedTeachingBubbleState = atom<boolean>({
   key: getFullyQualifiedKey('showGetStartedTeachingBubbleState'),
   default: false,
 });
+
+export const showAuthDialogState = atom<boolean>({
+  key: getFullyQualifiedKey('showAuthDialog'),
+  default: false,
+});
+
+export const primaryTokenState = atom<string>({
+  key: getFullyQualifiedKey('primaryToken'),
+  default: '',
+});
+
```

Comment:
As I mentioned in the last meeting, it would be great to have the expiration date/time of the token to reference.  You could have a selector for isTokenExpired (or isTokenValid) as a convenience.

Reply:
the expiration timestamp is now part of the currentUser object!